src/source/temporalgrid_tile_worker_source.js

```diff
@@ -47,23 +50,26 @@ const getAggregationParams = params => {
     return aggregationParams
 };
 
-const getFinalurl = (originalUrlString, { singleFrame, interval }) => {
+const getFinalurl = (originalUrlString, { singleFrame, interval, proxy = true }) => {
     const originalUrl = new URL(originalUrlString);
 
     const finalUrl = new URL(originalUrl.origin + originalUrl.pathname)
-
+    finalUrl.searchParams.append('proxy', proxy);
     finalUrl.searchParams.append('format', 'intArray');
-    if (originalUrl.searchParams.get("date-range")) {
-        finalUrl.searchParams.append('date-range', decodeURI(originalUrl.searchParams.get("date-range")))
-    }
     finalUrl.searchParams.append('temporal-aggregation', singleFrame);
     if (interval) {
         finalUrl.searchParams.append('interval', interval);
     }
+    const dateRange = originalUrl.searchParams.get("date-range")
+    if (dateRange) {
+        finalUrl.searchParams.append('date-range', decodeURI(dateRange))
+    }
+    const filters = originalUrl.searchParams.get("filters")
+    if (filters) {
+        finalUrl.searchParams.append('date-range', decodeURI(filters))
```

Comment:
```suggestion
        finalUrl.searchParams.append('filters', decodeURI(filters))
```
(I guess it was a typo)